### PR TITLE
<Box/> - story improvement according to new guidelines

### PR DESCRIPTION
**Pull request by Zivilekin**

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6945/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6945

File: packages/wix-style-react/src/Box/docs/examples.js [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+<StorybookComponents.Stack>
+  <Box padding="SP6" margin="SP1" border="1px solid" borderColor="D50" borderRadius={0}></Box>
+  <Box padding="SP6" margin="SP1" border="2px dotted" borderColor="D40" borderRadius={6}></Box>
+  <Box padding="SP6" margin="SP1" border="3px dashed" borderColor="D30" borderRadius={12}></Box>
+  <Box padding="SP6" margin="SP1" border="6px solid" borderColor="D20" borderRadius={72}></Box>
+</StorybookComponents.Stack>;
+`;
+
+export const inline = `
+<Layout cols={1}>
```
Comment (mykas) [inline packages/wix-style-react/src/Box/docs/examples.js:142]: Here it should be StorybookComponents.Stack instead.

File: packages/wix-style-react/src/Box/docs/examples.js [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+    This is a sentence with an action at the end of it.
+    <Box inline>
+      <TextButton>Link added in an inline box</TextButton>
+    </Box>
+  </Text>
+</Layout>;
+`;
+
+export const eventItem = `
+<Box minHeight={200} dataHook="storybook-multiple-boxes-within-box">
```
Comment (mykas) [inline packages/wix-style-react/src/Box/docs/examples.js:159]: This example does not follow snippet guidelines principles. There are a lot of unnecessary props that are not related to the example like for example: `dataHook`.

File: packages/wix-style-react/src/Box/docs/examples.js [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+  </Box>
+</Box>;
+`;
+
+export const listItems = `
+<TableListItem
+  options={[
+    {
+      value: (
+        <Box verticalAlign="middle">
```
Comment (mykas) [inline packages/wix-style-react/src/Box/docs/examples.js:201]: This example does not follow snippet guidelines principle on when there is a huge prop used - we should use functional snippet + variables: https://github.com/justinara/story_guidelines/blob/master/storybookguidelines.md#3-in-case-property-requires-large-data-sets-for-interpretation---functional-snippets-can-be-used-to-move-big-data-sets-to-meaningful-variables

**Review by mykas — APPROVED** [review]
Good job, @Zivilekin !
